### PR TITLE
Test tidying

### PR DESCRIPTION
**Pull request by drewnoakes**
- Reinstate commented test
- Use assertions that provide more details when they fail

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs [hunk 1]
```diff
@@ -247,41 +247,38 @@ public void Dependency_SetProperties_PreservesDependencyIDs()
...
-        //    };
-
-        //    mockSnapshot.Setup(x => x.DependenciesWorld).Returns(
-        //        ImmutableDictionary<string, IDependency>.Empty.AddRange(children));
-
-        //    Assert.True(dependency1.Dependencies.Count() == 2);
-        //    Assert.True(dependency1.Dependencies.Any(x => x.Equals(dependency2)));
-        //    Assert.True(dependency1.Dependencies.Any(x => x.Equals(dependency3)));
-        //}
+        [Fact]
```
Comment (etbyrd) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs]: I wonder why this was commented out here: https://github.com/dotnet/project-system/commit/c9f4b6c943a04c0b9df1928824327a4951fff263#diff-da666934981933a9cc43c6fdaf1b6d4aL227

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs]: I can't tell from context. Perhaps @abpiskunov can comment when he's back in the office.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs [hunk 2]
```diff
@@ -234,7 +234,7 @@ public void Dependency_SetProperties_PreservesDependencyIDs()
             var expectedDependencyId = "tfm1\\providerType\\glass";
 
             Assert.Equal(expectedId, dependency.Id);
-            Assert.True(dependency.DependencyIDs.Count == 1);
+            AssertEx.CollectionLength(dependency.DependencyIDs, 1);
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs]: Nit: You used `.Single` for this case elsewhere. I have no idea which gives a better message though ¯\\\_(ツ)_/¯

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyTests.cs]: Thanks. Fixed.

**Pull request merged**